test.js

```diff
@@ -0,0 +1,7 @@
+const Gls = require("require("./dist/commonjs/Gls.js")".Gls;
+
+const gls = new Gls();
+gls.setLanguage("CSharp");
+
+// System.Console.WriteLine("Hello world!");
+console.log(gls.convert([`print : ("Hello world!")`]));
```

Comment:
Did you force-add this file? There shouldn't be any `.js` files checked in except for test cases.